vrviz_ros/Topic[] topic_list
geometry_msgs/TransformStamped[] frame_list
visualization_msgs/Marker[] mesh_list